sql/csv/number-of-households-by-usage-of-cooking-fuel.csv

```diff
@@ -0,0 +1,6705 @@
+main type of cooking fuel,total,geo_code,geo_level
+Bio-Gas Fuel,1757165,NP,country
+Cow Dung,10197173,NP,country
+Electricity Fuel,53387,NP,country
+KeroseneFuel,773922,NP,country
+LP Gas,14205873,NP,country
+Not Sated about Fuel,514025,NP,country
```

Comment:
This should be "Not Stated" instead of "Not Sated"

Reply:
Fixed.